cli/provider_cache.go

```diff
@@ -233,6 +233,7 @@ func runTerraformCommand(ctx context.Context, opts *options.TerragruntOptions, a
 	cloneOpts.ErrWriter = errWriter
 	cloneOpts.WorkingDir = opts.WorkingDir
 	cloneOpts.TerraformCliArgs = args
+	cloneOpts.IncludeModulePrefix = false
```

Comment:
To avoid similar issue in the future, it's better to improve the regular expression itself instead. Since I was unable to reproduce this issue locally (this may depend on the shell and its settings) I will ask you to test a different approach. Please revert these changes and replace line 38 with:

```go
HTTPStatusCacheProviderReg = regexp.MustCompile(`(?smi)` + strconv.Itoa(controllers.HTTPStatusCacheProvider) + `.*` + http.StatusText(controllers.HTTPStatusCacheProvider))
```

Reply:
@levkohimins yes it does help as well. However I think we generally do not want to prepend module prefix in provider terraform commands as they are for nothing? I'm going to adjust regex but keep original commit. If you insist on removing that IncludeModulePrefix=false, let me know and I'll do so.

Reply:
I just wanted to make sure that just replacing the regex would also fix this issue (without your changes). Thanks you!